### PR TITLE
[WIP]Add Import Yaml Feature

### PR DESCRIPTION
**Pull request by orangedeng**
Related issue: https://github.com/rancher/rancher/issues/12627
Types pr: https://github.com/rancher/types/pull/377
Norman pr: https://github.com/rancher/norman/pull/142
1. Import action for cluster
2. Import action for project

### CODE REVIEW

File: pkg/api/customization/cluster/formatter.go [hunk 1]
```diff
@@ -56,3 +72,47 @@ func (a ActionHandler) GenerateKubeconfigActionHandler(actionName string, action
...
+	if err != nil {
+		return err
+	}
+	msg, err := kubectl.Apply([]byte(input.Yaml), cfg)
+	if err != nil {
+		return err
+	}
+
+	rtn := map[string]interface{}{
+		"outputMassage": string(msg),
```
Comment (ibuildthecloud) [inline pkg/api/customization/cluster/formatter.go]: typo: message

**Comment by orangedeng** [comment]
@ibuildthecloud Updated

**Comment by ibuildthecloud** [comment]
Included in https://github.com/rancher/rancher/pull/12829

**Comment by orangedeng** [comment]
I got your point. The only reason I do this is I need to get a cluster id in a project action. The way I saw from other action is from subcontext. is it other way to do it? or it just does not recommend to do that as a project action.

**Pull request closed without merging**